full-populate.sh

```diff
@@ -21,10 +21,18 @@ echo 'Creating index: Related'
 curl -s -XPUT $HOST/related/ -H "Content-Type: application/json" -d @/schemas/related-index.json
 echo 'Done creating index: Related'
 
+echo 'Creating index: Metrics'
+curl -s -XPUT $HOST/metrics/ -H "Content-Type: application/json" -d @/schemas/metrics-index.json
+echo 'Done creating index: Metrics'
+
 echo 'Adding data to index: DataAssets'
 elasticdump --input=/data/dataassets-data.json --output=$ELASTICSEARCH_URL/dataassets --type=data
 echo 'Done adding data to index: DataAssets'
 
 echo 'Adding data to index: Related'
 elasticdump --input=/data/related-data.json --output=$ELASTICSEARCH_URL/related --type=data
-echo 'Done adding data to index: Related'
\ No newline at end of file
+echo 'Done adding data to index: Related'
+
+echo 'Adding data to index: Metrics'
+elasticdump --input=/data/metrics-data.json --output=$ELASTICSEARCH_URL/metrics --type=data
+echo 'Done adding data to index: Metrics'
```

Comment:
Can you please put a new line at the end of this file so it isn't marked as a smell?